api/wallet_test.go

```diff
@@ -264,6 +264,15 @@ func TestIntegrationWalletTransactionGETid(t *testing.T) {
 	if len(wtg.UnconfirmedTransactions) != 0 {
 		t.Error("expecting 0 unconfirmed transactions")
 	}
+	// A call to /wallet/transactions without startheight and endheight parameters
+	// should return a descriptive error message.
+	err = st.getAPI("/wallet/transactions", &wtg)
+	if err == nil {
+		t.Error("expecting /wallet/transactions call with empty parameters to error")
+	}
+	if err.Error() != "startheight and endheight must be provided to a /wallet/transactions call.\n" {
```

Comment:
this will still panic if `err == nil`, because `t.Error` doesn't stop execution. Prefer:
```go
if err == nil || err.Error() != "blah blah blah" {

}
```